docs/guide/repository.md

```diff
@@ -1549,6 +1549,124 @@ class AcmeFooBundle extends Bundle
 
 Policies used internally in repository services are defined in  `EzPublishCoreBundle/Resources/config/policies.yml` .
 
+### Custom Limitation UI
+
+To provide support for editing custom policies in Platform UI you need to implement [`EzSystems\RepositoryForms\Limitation\LimitationFormMapperInterface`](https://github.com/ezsystems/repository-forms/blob/master/lib/Limitation/LimitationFormMapperInterface.php):
+
+```php
+<?php
+
+/**
+ * Interface for LimitationType form mappers.
+ *
+ * It maps a LimitationType's supported values to editing form.
+ */
+interface LimitationFormMapperInterface
+{
+    /**
+     * Maps Limitation form to current LimitationType, in order to display one or several fields
+     * representing limitation values supported by the LimitationType.
+     *
+     * Implementors MUST either:
+     * - Add a "limitationValues" form field
+     * - OR add field(s) that map to "limitationValues" property from $data.
+     *
+     * @param FormInterface $form Form for current Limitation.
+     * @param Limitation $data Underlying data for current Limitation form.
+     */
+    public function mapLimitationForm(FormInterface $form, Limitation $data);
+    /**
+     * Returns the Twig template to use for rendering the Limitation form.
+     *
+     * @return string
+     */
+    public function getFormTemplate();
+    /**
+     * This method will be called when FormEvents::SUBMIT is called.
+     * It gives the opportunity to filter/manipulate Limitation values.
+     *
+     * @param Limitation $limitation
+     */
+    public function filterLimitationValues(Limitation $limitation);
+}
+```
+
+Next, register the service in DIC (Dependency Injection Container) with the `ez.limitation.formMapper` tag and set the `limitationType` attribute to the Limitation type's identifier:
+
+```yml
+acme.security.limitation.custom_limitation.mapper:
+    class: 'Acme\FooBundle\Security\Limitation\Mapper\CustomLimitationFormMapper'
```

Comment:
Sorry for inconsistency. This should be `AppBundle\Security\Limitation\Mapper\CustomLimitationFormMapper` instead of `'Acme\FooBundle\Security\Limitation\Mapper\CustomLimitationFormMapper'`